src/main/java/com/gbtec/gameoflife/custom/GameOfLifeService.java

```diff
@@ -0,0 +1,47 @@
+package com.gbtec.gameoflife.custom;
+
+import com.gbtec.gameoflife.basic.GameOfLifeCommandProxy;
+
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class GameOfLifeService extends GameOfLifeCommandProxy {
+
+    public GameOfLifeService(SimpMessagingTemplate simpMessagingTemplate) {
+        super(simpMessagingTemplate);
+    }
+
+    @Override
+    public void init() {
+        // Instead of "int[][]" you can also use "boolean[][]"
+        int[][] generationData = {
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0},
+                {0, 0, 1, 0, 1, 0, 0, 0},
+                {1, 0, 0, 0, 0, 0, 1, 0},
+                {0, 1, 0, 0, 0, 1, 0, 0},
+                {0, 0, 1, 1, 1, 0, 0, 0},
+                {0, 0, 0, 0, 0, 0, 0, 0}
+        };
+
+        drawGeneration(generationData);
+
+        /* To generate random values use class "Random"
+         *
+         * Random random = new Random();
+         * random.nextBoolean(); // Generates "true" or "false"
+         * random.nextInt(0, 2); // Generates "0" or "1"
+         */
+    }
+
+//    @Override
+//    public void next() {}
+
+//    @Override
+//    public void play(int delayMs) {}
+
+//    @Override
+//    public void stop() {}
```

Comment:
Do not comment code out - rather provide a skeleton like the following:
```suggestion
    @Override
    public void next() {
        // TODO: implement
    }

    @Override
    public void play(int delayMs) {
        // TODO: implement
    }

    @Override
    public void stop() {
        // TODO: implement
    }
```

Reply:
The problem with this is, that in the parent class i have some implementation and if i provide the skeleton like suggested, given implementation will be overridden. I could just remove those (commented out) functions, but then a beginner who is not familiar with inheritance will probably not find the available (call back) methods.

Reply:
Ok, next suggestion: just put a comment into the readme to override the methods next, play, stop, init. The intern will then either ask questions how it works (which is a good thing) or find it out by herself (after doing the Java course)..